python/mxnet/module/module.py

```diff
@@ -18,6 +20,8 @@
 
 from .base_module import BaseModule, _check_input_names, _parse_data_desc
 
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
If I don't do this, docs build fails after adding link as follows:
:meth:`BaseModule.backward`.